po/pl.po

```diff
@@ -360,26 +353,30 @@ msgid "%s is inaccessible and can't be created\n"
 msgstr "%s jest niedostępny i nie może zostać utworzony\n"
 
 #: src/controller.cpp:295
-#, fuzzy
 msgid "ERROR: You must set `miniflux-url` to use Miniflux\n"
-msgstr "BŁĄD: Musisz ustawić `cookie-cache` aby korzystać z NewsBlur.\n"
+msgstr "BŁĄD: Musisz ustawić `miniflux-url` aby korzystać z Miniflux.\n"
 
 #: src/controller.cpp:307
 msgid ""
 "ERROR: You must set `miniflux-login` and one of `miniflux-password`, "
 "`miniflux-passwordfile` or `miniflux-passwordeval` to use Miniflux\n"
 msgstr ""
+"BŁĄD: Musisz ustawić `miniflux-login` oraz jeden z `miniflux-password`, "
+"`miniflux-passwordfile` olub`miniflux-passwordeval` aby korzystać z "
```

Comment:
I think a space is missing here:

```suggestion
"`miniflux-passwordfile` olub `miniflux-passwordeval` aby korzystać z "
```

Reply:
And a needles `o` to start with :facepalm: